CHANGES.md

```diff
@@ -1,11 +1,300 @@
 Release Changes List
 ====================
 
+version 1.27.0
+==============
+
+Thirtieth public release of Chapel, June 30, 2022
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* improved GPU support, including enabling the use of multiple locales
+* improved namespace behavior w.r.t. `import`, `use`, and shadowing
+* the Chapel compiler now supports LLVM 14, in addition to 11, 12, and 13
+* improved the performance and scalability of scan operations on `Block` arrays
+* added new 'OS.POSIX' and 'GPUDiagnostics' modules
+* restored support for Omni-Path networks
+* introduced a Code of Conduct for Chapel developers
+* made many changes, mostly minor, in support of Chapel 2.0 stabilization
+* many other bug fixes and improvements to error messages and documentation
+
+Packaging / Configuration Changes
+---------------------------------
+* the compiler's LLVM back-end now supports version 14 as well as 11, 12, 13
+* made Chapel available as an Arch Linux AUR package  
+  (see https://aur.archlinux.org/packages/chapel)
+* updated `configure` to create prefix directories if they don't already exist
+* `make install` now installs `chpldoc` if it is built
+
+Semantic Changes / Changes to Chapel Language
+---------------------------------------------
+* made small integers prefer coercions to `real(32)` over `real(64)`  
```

Comment:
coercions or coercing? (unsure myself)

Reply:
I think either could work but I'm going to call this under my bar for today.  :)